addons/index.md

```diff
@@ -8,15 +8,50 @@ layout: documentation
 
 All add-ons for openHAB 2 are part of the distribution.
 This includes all new 2.0 bindings as well as all 1.x add-ons that were reported to be compatible.
-Add-ons can be easily installed through the Paper UI (menu entry "Add-ons").
-
-
-| Add-on Type | Description |
-|-------------|----------------------|
-| [Bindings](bindings.html) | Bindings integrate physical hardware, external systems and web services in openHAB |
-| [User Interfaces](uis.html) | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
-| [Persistence](persistence.html) | Persistence services allow openHAB to store time series data for history-based actions or statistics |
-| [Actions](actions.html) | Actions are predefined methods for openHAB rules and scripts |
-| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items |
-| [Voice Services](voices.html) | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc. |
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+Add-ons can be easily installed in different ways, which are described below in detail.
```

Comment:
@bgilmer77 is this sentence in the correct order?

Reply:
i think installed easily is the correct order.

Reply:
I was actually talking about "are described in detail below." :)

Reply:
@ThomDietrich it is a little unclear.  Case 1) some of the add-ons might have to be installed differently from others.  Case 2) ALL add-ons may be installed in several ways.  Since we know Case 2 is the way it is, please consider the following:  "There are several ways you can install an add-on.  These are described under "Installation of Add-ons" below.

Reply:
👍